src/components/catagories.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/button-has-type */
```

Comment:
- [ ] Please don't disable linters. They are there to help you adhere to strict guidelines 😎